service/provider/src/main/java/gov/va/vro/service/provider/mas/service/mapper/MasCollectionAnnotsResults.java

```diff
@@ -119,27 +133,52 @@ public AbdEvidence mapAnnotationsToEvidence(MasCollectionAnnotation masCollectio
     return abdEvidence;
   }
 
-  private static AbdBloodPressure createBloodPressure(MasAnnotation masAnnotation) {
-    String[] bpValues = masAnnotation.getAnnotVal().split("/");
+  private static AbdBloodPressure createBloodPressure(MasAnnotation masAnnotation)
+      throws MasException {
+    log.info("MasAnnotation: {}", masAnnotation.getAnnotVal());
+    // Pattern to handle - "120/80 mg/dL"; // or "120/80" or "-/80" or "120/-"
+    Pattern pattern = Pattern.compile(MAS_BP_READING_VERIFICATION);
+    Matcher matcher = pattern.matcher(masAnnotation.getAnnotVal());
+    if (!matcher.matches()) {
```

Comment:
Could you please check with the business team to see whether an exception has to be raised if the BP annotation values don't match the expected format.